cookbook-efforts/dpo-orpo-preference/README.md

```diff
@@ -8,53 +8,55 @@ Currently, many languages do not have DPO datasets openly shared on the Hugging
 
 The goal of this project is to help foster a community of people building more DPO datasets for different languages.
 
-## tl;dr what is the output of this project?
+## Introduction
+
+### What are the objectives of this project?
```

Comment:
this might need to Why what how etc might be a bit more aligned with the other cookbook efforts?